src/twisted/conch/ssh/keys.py

```diff
@@ -322,7 +317,7 @@ def _fromString_PUBLIC_OPENSSH(cls, data):
         @rtype: L{twisted.conch.ssh.keys.Key}
         @raises BadKeyError: if the blob type is unknown.
         """
-
+        #decoding and splitting messes up ecdsa keys.
```

Comment:
Ok, so the reasoning for why seems to be that `decodebytes` is actually doing a base64 decode, which is not what is expected for ECDSA keys. It'd be nice if we could say that more clearly.